### PR TITLE
Fixed storage.total_limit_size misleading document content

### PR DESCRIPTION
**Pull request by opencmit2**
Fixed storage.total_limit_size misleading document content

image

### CODE REVIEW

File: administration/buffering-and-storage.md [hunk 1]
```diff
@@ -191,5 +191,5 @@ The following example create records with CPU usage samples in the filesystem an
     storage.total_limit_size  5M
 ```
 
-If for some reason Fluent Bit gets offline because of a network issue, it will continue buffering CPU samples but just keep a maximum of 5M of the newest data.
+If for some reason Fluent Bit gets offline because of a network issue, it will continue buffering CPU samples but just keep a maximum of 5M bytes of the newest data.
```
Comment (lecaros) [inline administration/buffering-and-storage.md:194]: please make the unit one word, i.e., MB or Megabytes

**Review by lecaros — CHANGES_REQUESTED** [review]

File: administration/buffering-and-storage.md (hunk 1 above)
Comment (opencmit2) [reply administration/buffering-and-storage.md]: i have modified it

**Comment by opencmit2** [comment]
Hi @lecaros 
now it has been dealt with, please review.

**Review by lecaros — APPROVED** [review]

**Comment by lecaros** [comment]
Merged! Thanks for your contribution @opencmit2

**Pull request merged**